### PR TITLE
Disable tests that are failing in CI on Unix

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

**Comment by jkotas** [comment]
@dotnet-bot test OSX x64 Checked Build and Test  please

**Comment by jkotas** [comment]
@wtgodbe @AndyAyersMS These two tests are failing in all CI jobs. How do I disable them until they are fixed? Adding them to testsFailingOutsideWindows.txt does not seem to have effect.

File: tests/testsFailingOutsideWindows.txt [hunk 1]
```diff
@@ -208,4 +208,5 @@ GC/Features/SustainedLowLatency/sustainedlowlatency_race_reverse/sustainedlowlat
 GC/Features/SustainedLowLatency/scenario/scenario.sh
 GC/Features/SustainedLowLatency/sustainedlowlatency_race/sustainedlowlatency_race.sh
 GC/Regressions/dev10bugs/536168/536168/536168.sh
-
+JIT/Regression/CLR-x86-EJIT/V1-M09.5-PDC/b14426/b14426.sh
+JIT/Performance/CodeQuality/BenchmarksGame/pidigits/pi-digits.sh
```
Comment (AndyAyersMS) [inline tests/testsFailingOutsideWindows.txt]: I think you need an extra `pi-digits` in there, eg 
```
JIT/Performance/CodeQuality/BenchmarksGame/pidigits/pi-digits/pi-digits.sh
```
probably similar fixup for the other test.

Is there an issue open so we can get the underlying problem fixed?

**Comment by wtgodbe** [comment]
Andy is correct - it should be 

`JIT/Regression/CLR-x86-EJIT/V1-M09.5-PDC/b14426/b14426/b14426.sh
JIT/Performance/CodeQuality/BenchmarksGame/pidigits/pi-digits/pi-digits.sh`

Since when the tests are built, each .csproj creates a folder inside the current folder with the name of the test - so if 'pi-digits.csproj' is inside a folder named pidigits, pi-digits.sh will be inside a folder named pi-digits inside of pidigits.

@AndyAyersMS I opened this issue to track these (https://github.com/dotnet/coreclr/issues/3664), I'll add these 2 to the list.

**Comment by jkotas** [comment]
I have fixed up the test paths. Thanks!

**Comment by gkhanna79** [comment]
These failures are being tracked by https://github.com/dotnet/coreclr/issues/3763

**Pull request merged**